stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.14
+version: 5.0.15
```

Comment:
Should be 5.0.18.

Reply:
Ok, but can you explain why to 5.0.18 and not 5.0.17 next version of the chart ?